_posts/2018-08-30-on-the-fly-transformations.md

```diff
@@ -0,0 +1,283 @@
+
+# Using the on-the-fly trajectory transformations in MDAnalysis
+
+On-the-fly transformations have been introduced in version 0.18.1 of MDAnalysis. This long awaited feature brings to MDAnalysis a whole new level of functionality, allowing for more efficient workflows when analyzing and visualizing simulation trajectories.
```

Comment:
It's not really long awaited.  It's OK to write that this is your GSoC project.

Reply:
This is work you did and you can say that.

Reply:
Can we link to docs from MDAnalysis as well? How much of this is documented in the code yet?

Reply:
Each PR that I have adds a topic to the transformations in the online docs. So only the merged PR's are actually in the docs

Reply:
What is already in the docs (you can link to the dev docs at https://www.mdanalysis.org/mdanalysis) and which PRs are still pending?

Reply:
Rotations and some translations are already in the docs. #1973 and #2038 are yet to be merged. Both include docs.